src/software/ai/navigator/path_planner/hrvo/brute_force_nearest_neighbor_search.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
```

Comment:
I think this file is more appropriate in the `software/geom/algorithms` folder

Reply:
Can also remove `brute_force_` from the file name, since it's also not included in the function name either.

Reply:
Done